tests/sentry/tasks/test_weekly_reports.py

```diff
@@ -620,3 +620,69 @@ def test_group_status_to_color_obj_correct_length(self):
 
         unique_enum_count = len(enum_values)
         assert len(group_status_to_color) == unique_enum_count
+
+    @mock.patch("sentry.analytics.record")
+    @mock.patch("sentry.tasks.weekly_reports.MessageBuilder")
+    def test_email_override_simple(self, message_builder, record):
```

Comment:
Another test case, where `send_email` is mocked to throw `AttributeError` which would validate that `deliver_reports.email_override.error` is correctly raised would be helful.